docs/design/event-streams.md

```diff
@@ -141,15 +142,70 @@ Modeling the event stream as a field of the request or response allows for [init
 to be implemented. If we directly returned or took a `Flow<T>` as the input or output type we would not be able to represent the initial request or response fields when present.
 
 
+#### Event Stream Error Representation
+
+Event stream unions may model exceptions that can appear on the stream. These exceptions are terminal messages that are intended to be surfaced to the client using
+idiomatic error handling mechansims of the target language. This means that in the model the possible (modeled) errors a consumer may see after the stream has started
+are part of the overall union that makes up the possible events. 
```

Comment:
**Style**: This sentence is slightly unclear. May I suggest:

"Thus the modeled errors a consumer may see during the stream are part of the overall union that makes up the possible events."